UndertaleModLib/Models/UndertaleSprite.cs

```diff
@@ -114,6 +114,42 @@ public class UndertaleSprite : UndertaleNamedResource, PrePaddedObject, INotifyP
         /// </summary>
         public int OriginY { get; set; }
 
+        /// <summary>
+        /// A <see cref="OriginX"/> wrapper than also sets horizontal origin of <see cref="V2Sequence"/>.
+        /// </summary>
+        /// <remarks>
+        /// This attribute is used only in UndertaleModTool and doesn't exist in GameMaker.
+        /// </remarks>
+        public int OriginXWrapper
```

Comment:
Hm, in order to not have anyone use originX and thus forgetting to update V2Sequence.OriginX as well, shouldn't OriginX just be a private attribute and renamed to something else while this gets then renamed to OriginX?

Reply:
Yeah, I'll consider that.